src/patterns/components/slideshow/base.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="u-bgGray u-posRelative Slideshow js-Slideshow u-pullSides1">
+  <h2 class="u-hiddenVisually">Fun facts about Aileen</h2>
+  <ul class="u-flex u-listUnstyled Slideshow-slides js-Slideshow-slides u-padEnds2"  id="Slideshow-slides">
+    {{#each drizzle.data.fun-facts.contents.aileen}}
+      <li class="Grid Grid--withGutter Grid--alignMiddle Slideshow-slide js-Slideshow-slide u-flexExpand u-flexJustifyCenter u-sm-flexJustifyStart u-padSides5 {{#if @first}}is-visible{{/if}}">
+        <div class="Grid-cell u-size1of2 u-sm-size1of4 u-lg-size1of5">
+          <div class="Thumbnail Thumbnail--circle u-block">
+            <img class="u-sizeFull" src="{{image}}" alt="">
+          </div>
+        </div>
+        <div class="Grid-cell u-sm-size3of4 u-lg-size1of2 u-spaceItems06 u-textCenter u-sm-textLeft u-spaceTop01 u-sm-spaceTopNone">
+          <h4 class="u-md-textGrow1">Did you know&hellip;?</h4>
+          <p>{{fact}}</p>
+        </div>
+      </li>
+    {{/each}}
+  </ul>
+  <ul class="Slideshow-nav u-listInline u-spaceSidesNone u-flexJustifyBetween u-flexAlignContentCenter u-lg-flexJustifyEnd u-lg-padRight3 u-posAbsoluteCenter u-size1of1" aria-controls="#Slideshow-slides">
+    <li class="u-spaceLeft05">
+      <a class="u-linkClean js-Slideshow-prev" href="">
```

Comment:
> Is the idea that these links will resolve to the ID of the next and previous, or to some server-side implementation?

Nope, I meant to swap this to a button, but forgot. I think I initially used a link since it has all the color/hover that I want, and then never went back to it. You're right that `button` is a better choice.